mgmt/RecordsConfig.cc

```diff
@@ -1995,6 +1995,8 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.http2.no_activity_timeout_in", RECD_INT, "115", RECU_DYNAMIC, RR_NULL, RECC_STR, "^[0-9]+$", RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.http2.active_timeout_in", RECD_INT, "900", RECU_DYNAMIC, RR_NULL, RECC_STR, "^[0-9]+$", RECA_NULL}
```

Comment:
I'm not sure how log is suitable for default value. I chose `900` to make it same as `proxy.config.http.transaction_active_timeout_in`.
Any suggestions?